.teamcity/_self/projects/WebApp.kt

```diff
@@ -614,6 +614,27 @@ object Translate : BuildType({
 			"""
 			dockerImage = "%docker_image_e2e%"
 		}
+		bashNodeScript {
```

Comment:
Jorn has already reviewed it, so just a nitpick - it looks like the code between lines 614 and 640 is not aligned with the rest and should be moved to the right.

Reply:
Are you referring to this section - https://github.com/Automattic/wp-calypso/blob/8c461bd29bfefbdaf8dfa996f24fb13301ee7243/.teamcity/_self/projects/WebApp.kt#L614-L640? It looks alright to me.

Reply:
If you'd open https://github.com/Automattic/wp-calypso/pull/61159/files and compare line 599 to 617, the two sections won't be aligned, but it looks like the actual file is properly formatted, so I'm not sure what's causing it.

<img width="1147" alt="Image 2022-02-17 at 2 13 30 AM" src="https://user-images.githubusercontent.com/7847633/154385231-45eef951-50b5-4090-a70a-3d6d527141bd.png">